components/Card/Card.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from "react";
+import styled from "styled-components";
+
+interface Props extends React.Attributes {
+  onClick?: React.MouseEventHandler<HTMLElement>;
+  className?: string;
+  style?: React.CSSProperties;
+  children?: React.ReactNode;
+}
+
+export default function Card({ children, ...props }: Props) {
+  return <Root {...props}>{children}</Root>;
+}
+
+const Root = styled.div<{
+  onClick?: React.MouseEventHandler<HTMLElement>;
+}>`
+  border-radius: 4px;
+  box-shadow: 0px 0px 12px #222f3e1f, 0px 12px 24px #222f3e0f;
+  display: block;
+  transition: box-shadow 200ms ease, transform 200ms ease;
+
+  ${({ onClick }) =>
+    onClick
+      ? `
+      &:hover {
+        cursor: pointer;
```

Comment:
`cursor: pointer;` should be outside of `:hover`. Because this property sets the cursor's general behavior when it's on the element.

ref. https://developer.mozilla.org/en-US/docs/Web/CSS/cursor

```suggestion
  cursor: pointer;
  transition: box-shadow 200ms ease, transform 200ms ease;

  ${({ onClick }) =>
    onClick
      ? `
      &:hover {
```